### PR TITLE
Burst rate limits

### PR DESCRIPTION
**Pull request by christinechoy-okta**
## Description:
- **What's changed?** Add burst rate limits
- **Is this PR related to a Monolith release?** 2022.02.0

### Resolves:

* [OKTA-459072](https://oktainc.atlassian.net/browse/OKTA-459072)

### CODE REVIEW

**Review by joec-okta — APPROVED** [review]
LGTM

File: packages/@okta/vuepress-site/docs/reference/rate-limits/index.md [hunk 1]
```diff
@@ -31,6 +31,25 @@ If any org-wide rate limit is exceeded, an HTTP 429 status code is returned. You
 > * Rate limits may be changed to protect customers. We provide advance warning of changes when possible.
 > * You can expand the Okta rate limits upon request. To learn how, see [Request exceptions](/docs/reference/rl-best-practices/#request-exceptions) and [DynamicScale rate limits](/docs/reference/rl-dynamic-scale/).
 >
+
+## Burst rate limits
+
+Okta provides rate limits for orgs based on the traffic that they expect to have. If your org experiences higher traffic than what is expected, this unplanned usage may potentially have an impact on end users. To ensure our customers are successful and to help minimize impact, Okta will not suspend usage above the rate limit (specifically for authentication and authorization flows). In the event that there is sustained usage above the purchased rate limit, Okta will require that you purchase an applicable offering that is in line with your usage. By making burst rate limits available, Okta provides peace of mind by ensuring an unplanned spike doesn't detrimentally affect the end user's experience.
```
Comment (hmobius) [inline packages/@okta/vuepress-site/docs/reference/rate-limits/index.md:37]:
```suggestion
Okta provides rate limits for orgs based on the traffic they expect to have. If your org experiences higher traffic than expected, this unplanned usage may potentially have an impact on end users. To ensure our customers are successful and to help minimize impact, Okta will not suspend usage above the rate limit (specifically for authentication and authorization flows). In the event of sustained usage above the purchased rate limit, Okta will require you purchase an applicable offering in line with your usage. By making burst rate limits available, Okta provides peace of mind by ensuring an unplanned spike doesn't detrimentally affect the end user's experience.
```

**Review by hmobius — CHANGES_REQUESTED** [review]
Just a couple of thats and whats that seem extraneous

**Review by spencermccoubrey-okta — APPROVED** [review]

File: packages/@okta/vuepress-site/docs/reference/rate-limits/index.md (hunk 1 above)
Comment (christinechoy-okta) [reply packages/@okta/vuepress-site/docs/reference/rate-limits/index.md:37]: @hmobius, this paragraph was written the way that it was due to:
1. Legal reasons and legal reviewers
2. Best practice for localization

**Review by susanharper-okta — COMMENTED** [review]
couple things - I have a love/hate relationship with jargon - and "usage" is one of those words. If we don't HAVE to use it and can use "use" instead, it makes it easier to read. I know it's in the UI, but if you aren't talking about the UI, can you use "use" instead? :)

File: packages/@okta/vuepress-site/docs/reference/rate-limits/index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (christinechoy-okta) [reply packages/@okta/vuepress-site/docs/reference/rate-limits/index.md:37]: I made all the updates except for the "usage">"use" change.

File: packages/@okta/vuepress-site/docs/reference/rate-limits/index.md [hunk 2]
```diff
@@ -31,6 +31,25 @@ If any org-wide rate limit is exceeded, an HTTP 429 status code is returned. You
...
+
+In a typical use case scenario, when orgs exceed a default rate limit, they receive a system log warning event, then a burst event, and then a violation event. For example, an org has a rate limit of 600 requests per minute on the `/api/v1/authn` endpoint. That org would receive a warning at 360 requests per minute (60%) of 600. That org would get a burst notification when the endpoint hits 600 requests per minute and then the violation event when it hits 3000 requests all in the same minute.
+
+Additionally, burst rate limits will typically apply on top of any rate limit increase that an org may have, such as [DyanmicScale](/docs/reference/rl-dynamic-scale/). For example, the default limit on `/api/v1/authn` is 600 requests per minute. If an org is expecting traffic to require 6000 requests per minute, the org would purchase DynamicScale 10x. The burst rate limit in this scenario would provide 5x coverage on top of the 6000 and ensure peace of mind for any unplanned spike in usage.
```
[the remark this answers is not in the document]
Comment (christinechoy-okta) [reply packages/@okta/vuepress-site/docs/reference/rate-limits/index.md:41]: I made the tense change but didn't make the "usage">"use" change.

**Comment by christinechoy-okta** [comment]
> couple things - I have a love/hate relationship with jargon - and "usage" is one of those words. If we don't HAVE to use it and can use "use" instead, it makes it easier to read. I know it's in the UI, but if you aren't talking about the UI, can you use "use" instead? :)

I ended up changing the word "usage" to "use". Makes sense in the content.

**Pull request merged**